worker-message-prioritization-distribution-container/pom.xml

```diff
@@ -149,10 +149,10 @@
                     <verbose>true</verbose>
                     <images>
 
-                        <!-- Build a RabbitMQ image with the shovel plugins enabled for testing purposes -->
+                        <!-- Build a RabbitMQ image -->
                         <image>
                             <alias>rabbitmq</alias>
-                            <name>${dockerWorkerFrameworkOrg}test-rabbitmq-with-shovel-plugin:${project.version}</name>
+                            <name>${dockerWorkerFrameworkOrg}test-rabbitmq:${project.version}</name>
                             <build>
```

Comment:
We can run the existing RabbitMQ image, we don't need one with the plugin anymore.